.github/workflows/ci.yml

```diff
@@ -698,72 +707,9 @@ jobs:
-        if: steps.filter.outputs.product-tests == 'true' || contains(github.event.pull_request.labels.*.name, 'tests:all') || contains(github.event.pull_request.labels.*.name, 'tests:all-product')
-        run: |
-          testing/bin/ptl suite run \
-            --suite ${{ matrix.suite }} \
-            --config config-${{ matrix.config }} \
-            --bind=off --logs-dir logs/ --timeout 2h \
-            --trino-jdk-version zulu_${{ matrix.jdk }}
-      - name: Product Tests (impacted-features)
+      - name: Enable impact analysis
         if: steps.filter.outputs.product-tests == 'false' && !contains(github.event.pull_request.labels.*.name, 'tests:all') && !contains(github.event.pull_request.labels.*.name, 'tests:all-product')
-        run: |
-          testing/bin/ptl suite run \
-            --suite ${{ matrix.suite }} \
-            --config config-${{ matrix.config }} \
-            --impacted-features impacted-features.log \
-            --bind=off --logs-dir logs/ --timeout 2h \
-            --trino-jdk-version zulu_${{ matrix.jdk }}
-      - name: Upload test logs and results
-        uses: actions/upload-artifact@v2
-        # Upload all test reports only on failure, because the artifacts are large
-        if: failure()
-        with:
-          name: result pt (${{ matrix.config }}, ${{ matrix.suite }}, ${{ matrix.jdk }})
-          path: |
-            testing/trino-product-tests/target/*
-            logs/*
-      - name: Upload test report
-        uses: actions/upload-artifact@v2
-        # Always upload the test report for the annotate.yml workflow,
-        # but only the single XML file to keep the artifact small
-        if: always()
-        with:
-          # Name prefix is checked in the `Annotate checks` workflow
-          name: test report pt (${{ matrix.config }}, ${{ matrix.suite }}, ${{ matrix.jdk }})
-          path: testing/trino-product-tests/target/reports/**/testng-results.xml
-          retention-days: ${{ env.TEST_REPORT_RETENTION_DAYS }}
-
-  azure-pt:
-    runs-on: ubuntu-latest
-    needs: build-pt
-    if: needs.build-pt.outputs.have_azure_secrets == 'true'
-    strategy:
-      fail-fast: false
-      matrix:
-        config:
-          - hdp3
-        suite:
-          - suite-azure
-        jdk:
-          - 11
-    timeout-minutes: 30
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          fetch-depth: 0 # checkout all commits, as the build result depends on `git describe` equivalent
-      - uses: actions/setup-java@v2
-        with:
-          distribution: 'zulu'
-          java-version: 11
-      - name: Product tests artifact
-        uses: actions/download-artifact@v2
-        with:
-          name: product tests and server tarball
-      - name: Fix artifact permissions
-        run: |
-          find . -type f -name \*-executable.jar -exec chmod 0777 {} \;
+        run: echo "PTL_OPTS=--impacted-features impacted-features.log" >> $GITHUB_ENV
       - name: Product Tests
```

Comment:
```suggestion
      - name: Product Tests - ${{ env.PTL_OPTS }}
```

Would this work?

Reply:
Yes, according to [the docs](https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability) the `env` context is available in the step name.

Reply:
But IMO it'll kinda look ugly, do we really need this if there's an explicit step before this that's either skipped or not?

Reply:
I'm fine as it is. The entire argument would look ugly indeed.